### ISSUE TITLE
unable to tab to #toolbar-personal

### ISSUE THREAD

**Issue by tkimnguyen** [plone/volto#5396]
Using Safari 17.1 (19616.2.9.11.7) on macOS Sonoma 14.1.1 (23B81) with the setting enabled to "Press Tab to highlight each item on a webpage", I see a rectangular highlight around items as I tab over the front page of plone.org and demo.plone.org (while logged in), but I never see it around the personal toolbar icon (the head and shoulders of a person in the bottom left corner). I do see over some items in the toolbar. At times I see nothing highlighted.

**Comment by davisagli** [comment]
It looks like `a` has focus styles but `button` doesn't

**Comment by davisagli** [comment]
Related: https://github.com/plone/volto/pull/5335

**Comment by Wagner3UB** [comment]
@davisagli You are right, this is, actually, the main problem when the subject is "x doesn't have focus". 90% of the cases the element is a button, per example the "toolbar-add" and the "toolbar-more" elements of the CMS toolbar. 

<img width="966" alt="Screenshot 2023-11-22 at 08 51 54" src="https://github.com/plone/volto/assets/60133113/e2f53420-616a-406a-960b-95c193b8f7e4">

**Comment by Wagner3UB** [comment]
Here, I create a very general rule for focus-visible that 'solves' those problems, but it is too general and needs to be rethought:

https://github.com/plone/volto/pull/4793/files#diff-195c007a89d0b1c256713424ffd390e959b4e8a258300ceccd02e1bb234017ad
